### PR TITLE
Update index.tsx

### PR DESCRIPTION
**Pull request by michaelzhang43**
Updated to reflect new forms

### CODE REVIEW

File: src/frontend/src/components/Manual/index.tsx [hunk 1]
```diff
@@ -482,7 +485,16 @@ class Manual extends React.Component {
...
+                charges. RecordSponge will also generate a Request form to Oregon
+                State Police. You will need to complete this form manually and mail 
+                a completed copy to Oregon State Police at
+              </p>
+              Oregon State Police, CJIS – Unit 11
+              </p> ATTN: SET ASIDE 
+            </p> P.O. Box 4395 
+          </p> Portland, OR 97208-4395 
+
+
```
Comment (hmarcks) [inline src/frontend/src/components/Manual/index.tsx:497]:
```suggestion
                charges. RecordSponge will also generate a Request form to Oregon
                State Police. You will need to complete this form manually and mail 
                a completed copy to Oregon State Police at:
                <span className="db pt2">
                  Oregon State Police, CJIS – Unit 11<br>
                  ATTN: SET ASIDE<br>
                  P.O. Box 4395<br>
                  Portland, OR 97208-4395<br>
                </span>
```

File: src/frontend/src/components/Manual/index.tsx [hunk 2]
```diff
@@ -507,6 +519,20 @@ class Manual extends React.Component {
...
+                Included in your expungement packet should be a form titled: 
+                "Oregon State Police REQUEST FOR SET ASIDE CRIMINAL RECORD CHECK." 
+                Fill out the sections titled "Other Names You are Known By," 
+                "Circuit or Municipal Court," and check the box corresponding to whether
+                 you are seeking an expungement for a conviction or only arresets. 
+              </p>
+                If you are seeking expungement of at least one conviction, you 
+              will need to include a check or money order made out to 
+              "Oregon State Police" for $33. 
+              </p>
```
Comment (hmarcks) [inline src/frontend/src/components/Manual/index.tsx:535]:
```suggestion
              <p className="mb3">
                If you are seeking expungement of at least one conviction, you 
                will need to include a check or money order made out to 
                "Oregon State Police" for $33.
              </p>
```

**Review by hmarcks — CHANGES_REQUESTED** [review]
Just a couple of markup fixes

**Comment by hmarcks** [comment]
@wittejm @KentShikama let me know if these suggestions look okay and I can commit them, thanks

**Comment by KentShikama** [comment]
@hmarcks Yup looks good

**Review by hmarcks — APPROVED** [review]

**Pull request merged**